tests/dags/test_task_view_type_check.py

```diff
@@ -40,7 +40,7 @@ def __call__(self):
         """ A __call__ method """
 
 
-def a_function(arg_x, arg_y):
+def a_function(_, __):
```

Comment:
I know it's simple but I don't like that and the purpose of this function is not obvious. If we want assert that only two arguments are passed then we can validate number of args or kwargs and rise `TypeError`.